bin/configure.sh

```diff
@@ -99,3 +99,15 @@ iconv -f IBM-1047 -t IBM-850 ${STATIC_DEF_CONFIG_DIR}/zss.ebcidic.yml > $STATIC_
 rm ${STATIC_DEF_CONFIG_DIR}/zss.ebcidic.yml
 chmod 770 $STATIC_DEF_CONFIG_DIR/zss.yml
 fi
+
+# Setup certificates
+PREFIX="ZWED_agent_https_"
```

Comment:
This new section (and the `instanceBaseUrls` above) needs to be conditional upon var `ZOWE_ZSS_SERVER_TLS=true/false` as seen in instance.env.
This should enable the user to quickly switch between https and http for backward  compatibility.

Reply:
Also, currently the ZSS ip address defaults to 127.0.0.1 and ignores all the default values in instance.env, which confuses users.
When TLS is true, I think it would be great to set ip address to external.

`if (tls && !ZWED_agent_https_ipAddresses) { export ZWED_agent_https_ipAddresses = $ZOWE_IP_ADDRESS }`

Reply:
https://github.com/zowe/zowe-install-packaging/pull/1996/commits/a11690f3af9b7dc88c6709e75b0e4b10ac94f78c